src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { css } from "emotion";
+
+type Props = {
+  children: React.ReactNode;
+  direction?: "row" | "column";
+  justify?: "flex-start" | "space-between"; // Extend as required
+  fullWidth?: boolean;
```

Comment:
This doesn't really relate to `Flex` and makes it more of a generic container which runs the risk of stretching the definition for this abstraction.

Reply:
`fullWidth` is a very common thing I use with flex, as we agreed that we should not use classNames, I feel it makes sense to add this here.

without it, this component would become pointless, as you would have no way of setting full width on flex components.

Reply:
do you have a suggestion for the `fullWidth` work around? or are we ok to keep this?